cartographer_ros/cartographer_ros/offline_node_main.cc

```diff
@@ -232,6 +247,12 @@ void Run(const std::vector<string>& bag_filenames) {
 #endif
 
   node.SerializeState(bag_filenames.front() + ".pbstream");
+  if (FLAGS_keep_running) {
+    ::ros::WallRate rate(10.);
```

Comment:
Nice. I had been trying to find if something like this exists, but my Google-fu failed me.